docs/docs-content/vm-management/vm-packs-profiles/create-vmo-profile.md

```diff
@@ -53,14 +47,85 @@ The **Virtual Machine Orchestrator** pack conveniently includes several componen
 
 8. Click **Confirm & Create**. 
 
-
 9. In the following screen, click **Next**. 
 
-
 10. Review the profile and click **Finish Configuration**.
 
+11. Apply the profile to your cluster. For more information, refer to [Create a Cluster](../../clusters/public-cloud/deploy-k8s-cluster.md). 
+
+</TabItem>
+
+<TabItem value="edge" label="Edge">
+
+1. Log in to [Palette](https://console.spectrocloud.com).
+
+2. Select **Profiles** in the left **Main Menu** and click the **Add Cluster Profile** button.
+
+3. Enter basic information for the profile: name, version if desired, and optional description.
+
+4. Select type **Add-on**, and click **Next**.
+
+5. In the following screen, click **Add New Pack**. 
+
+6. Locate the **Virtual Machine Orchestrator** pack and add it to your profile. 
+
+7. Review the **Access** configuration panel at right. The default setting is **Proxied**, which automatically adds the **Spectro Proxy** pack when you create the cluster, allowing access to the Spectro VM Dashboard from anywhere. Check out the [Spectro Proxy](../../integrations/frp.md) guide to learn more. Changing the default may require some additional configuration. 
```

Comment:
```suggestion
7. Review the **Access** configuration panel at right. The default setting is **Proxied**, which automatically adds the **Spectro Proxy** pack when you create the cluster, allowing access to the Spectro VM Dashboard from anywhere.  Changing the default may require some additional configuration. Check out the [Spectro Proxy](../../integrations/frp.md) guide to learn more
```
It's better to end with the link, versus adding information after the link out. -